src/main/java/com/snowflake/kafka/connector/SchematizationUtils.java

```diff
@@ -55,7 +63,9 @@ public static Map<String, String> getAvroSchemaFromSchemaRegistryClient(
     try {
       schemaMetadata = schemaRegistry.getLatestSchemaMetadata(subjectName);
     } catch (IOException | RestClientException e) {
-      throw SnowflakeErrors.ERROR_0012.getException(e);
+      // suppress the excpetion and return empty map to indicate a failure in fetching schemas
+      LOGGER.info("Schema with subject '{}' not found in schema registry", subjectName);
```

Comment:
How often are we printing this?

Reply:
I believe every time we try to get the schema from schema registry but found nothing there.

Reply:
This be too much if someone doesn't opt into schematization